### ISSUE TITLE
Bugs on the application?

### ISSUE THREAD

**Issue by jeffryhartanto** [superjxvv/CS2102_2021_S1_43#4]
Whenever I submit bid with Dropoff at caretaker or make payment with cash, the app keeps loading and gives Heroku application error.

Also, when I edit the particulars of a pet, they won't get changed on the profile of the pet.

username: admin@gmail.com password: admin.

Any comment on these issues?

**Comment by jlxw48** [comment]
1. Previous error was due to query.rows not being able to be directly accessed, need to access rows separately after getting query. 
Previously, we used 
`pool.query(`SELECT address FROM care_taker WHERE email = $1`, [req.body.ct_email]).rows[0].address` 
which did not allow us to access the data returned from the SQL query. We have since rectified this bug.
2. We cannot reproduce the loading error while making a payment with cash from the Transactions page. To clarify, it is not our intention to change the transaction status through making a payment with cash from the Pet Owner's account. The CareTaker will have to confirm it on his side that he has received the cash payment.
3. There was an overlap in the URL parameters in the POST request `/add_pet/:action'`, which resulted in taking the wrong if-else clauses. We have split this up and also separated the SQL queries for further clarity to completely avoid this.
